src/core/dbus-cgroup.c

```diff
@@ -826,6 +829,31 @@ int bus_cgroup_set_property(
 
                 return 1;
 
+        } else if (STR_IN_SET(name, "MemoryLow", "MemoryHigh", "MemoryMax")) {
```

Comment:
Hm, something went wrong:
```sh
-bash-4.3# uname -r
4.4.0-22-generic

-bash-4.3# systemd-run --property MemoryMax=200M -r /bin/date
Running as unit: run-r31aca319054a476ca95c41b6da1f80d1.service

# note MemoryMax=209715200[Service]
-bash-4.3# systemctl cat run-r31aca319054a476ca95c41b6da1f80d1.service --no-pager
# /run/systemd/transient/run-r31aca319054a476ca95c41b6da1f80d1.service
# This is a transient unit file, created programmatically via the systemd API. Do not edit.
[Unit]
Description=/bin/date
[Service]
MemoryMax=209715200[Service]
RemainAfterExit=yes
[Service]
ExecStart=
ExecStart=@/bin/date "/bin/date"
```
I think we should print warnings on older kernels. Something like: `MemoryMax is not supported. Ignoring`